PolyEngine/RenderingDevice/OpenGL/Src/GLRenderingDevice.hpp

```diff
@@ -103,6 +106,7 @@ namespace Poly
 		EnumArray<std::unique_ptr<RenderingPassBase>, ePostprocessRenderPassType> PostprocessRenderingPasses;
 
 		std::unique_ptr<PostprocessQuad> PostprocessRenderingQuad;
+		std::unique_ptr<PrimitiveCube> PrimitiveRenderingCube;
```

Comment:
Maybe we could create cube with reversed UV to address fliped textures problem

Reply:
Better fix the vertex order in triangles.

Reply:
Vertex order used but it was not very effective.
Scaled UV by vec3(-1.0, 1.0, 1.0) and it was supper effective.
*Skybox flipped*